frontends/p4/tableKeyNames.cpp

```diff
@@ -81,6 +106,12 @@ class KeyNameGenerator : public Inspector {
         name.emplace(expression, e0 + "[" + e1 + ":" + e2 + "]");
     }
 
+    /** Compute a name annotation if @expression is a method call for
+     * `isValid()`.
+     *
+     * @warning All other method calls in keys will cause the compiler to
```

Comment:
The comment is inaccurate; the compiler will not reject other method calls; this method will invoke an error if it is invoked with another method call. However, if the key already has a name annotation, the compiler will not reject the program.